lib/keycloak-admin/client/token_client.rb

```diff
@@ -25,5 +31,15 @@ def get
       end
       TokenRepresentation.from_json(response.body)
     end
+
+    def revoke(user_id) 
```

Comment:
The name "remove" on the Token client is, imho, misleading.
Here, you are not only revoking a token, you are removing all user sessions, according to https://www.keycloak.org/docs-api/15.0/rest-api/index.html

Could you adapt your PR to make sure it is more aligned with the Keycloak REST Admin documentation?